src/routes/analytics/new/NewFromScratchPage.tsx

```diff
@@ -1,19 +1,29 @@
 import { Container } from "@mantine/core"
-import { InteractiveQuestion } from "@metabase/embedding-sdk-react"
+import {
+  InteractiveQuestion,
+  MetabotQuestion,
+} from "@metabase/embedding-sdk-react"
 
 import { useCreateQuestionHelpers } from "../../../utils/use-create-question-helpers"
+import { useSearchParams } from "wouter"
 
 export const NewFromScratchPage = () => {
   const { collectionId, onSaveQuestion } = useCreateQuestionHelpers()
+  const [searchParams] = useSearchParams()
+  const shouldShowMetabot = searchParams.get("metabot") === "true"
 
   return (
     <Container w="100%">
-      <InteractiveQuestion
-        questionId="new"
-        onSave={onSaveQuestion}
-        targetCollection={collectionId}
-        isSaveEnabled
-      />
+      {shouldShowMetabot ? (
+        <MetabotQuestion />
```

Comment:
This is the same markup as the metabot branch
https://github.com/metabase/shoppy/blob/434e9388d30185389730dbac787a985b068d6dd2/src/routes/analytics/new/NewFromScratchPage.tsx#L6-L8